src/System.Private.CoreLib/shared/System/Decimal.DecCalc.cs

```diff
@@ -990,8 +990,7 @@ internal static unsafe void DecAddSub(ref DecCalc d1, ref DecCalc d2, bool sign)
 
                     // Have to scale by a bunch. Move the number to a buffer where it has room to grow as it's scaled.
                     //
-                    Buf24 bufNum;
-                    _ = &bufNum; // workaround for CS0165
+                    Unsafe.SkipInit(out Buf24 bufNum); // workaround for CS0165
```

Comment:
I do not think we need the " workaround for CS0165" comments anymore. `Unsafe.SkipInit` is self-documenting.

Reply:
Removed comments in the latest commit.